WordPress/src/main/java/org/wordpress/android/ui/photopicker/PhotoPickerViewModel.kt

```diff
@@ -97,8 +96,8 @@ class PhotoPickerViewModel @Inject constructor(
                 softAskRequest?.show == true,
             ),
             buildSoftAskView(softAskRequest),
-            FabUiModel(browserType.isWPStoriesPicker && selectedIds.isNullOrEmpty()) {
-                clickIcon(PhotoPickerFragment.PhotoPickerIcon.WP_STORIES_CAPTURE)
+            FabUiModel(selectedIds.isNullOrEmpty()) {
+                clickIcon(PhotoPickerFragment.PhotoPickerIcon.ANDROID_CAPTURE_PHOTO)
```

Comment:
I am not confident with this change. Please double check this.

Reply:
I think we may remove the ` FabUiModel` completely 🤔 
I tried this in f4a25d1e0d0eb293c55e8d61f8090ddc7cf82ce6